src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DoNotUseStackallocInLoopsTests.cs

```diff
@@ -63,6 +63,26 @@ unsafe class TestClass {
                 }");
         }
 
+        [Fact]
+        public async Task NoDiagnostics_StackallocInLoopButInsideALocalFunction()
+        {
+            await VerifyCS.VerifyAnalyzerAsync(@"
```

Comment:
You will have to use the full syntax to specify C# 8 version in order to use the local function feature for your test.